http/multipart/multipart.go

```diff
@@ -0,0 +1,57 @@
+package multipart
```

Comment:
We should look for a new name that will not conflict with go official "mime/multipart" library. When both are used, we will have to add alliance to imports (normal scenario).